packages/perseus-core/src/widgets/matcher/matcher-util.ts

```diff
@@ -0,0 +1,31 @@
+import type {PerseusMatcherWidgetOptions} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusMatcherWidgetOptions type
+ */
+type MatcherPublicWidgetOptions = {
+    labels: PerseusMatcherWidgetOptions["labels"];
+    left: Array<string>;
+    right: Array<string>;
+    orderMatters: PerseusMatcherWidgetOptions["orderMatters"];
+    padding: PerseusMatcherWidgetOptions["padding"];
+};
+
+/**
+ * Given a PerseusMatcherWidgetOptions object, return a new object with only
+ * the public options that should be exposed to the client.
+ */
+function getMatcherPublicWidgetOptions(
+    options: PerseusMatcherWidgetOptions,
+): MatcherPublicWidgetOptions {
+    return {
+        labels: options.labels,
+        left: options.left.slice().sort(),
+        right: options.right.slice().sort(),
```

Comment:
Thoughts on a different/better way to remove order information?

Reply:
🤔 So we [shuffle these options](https://github.com/Khan/perseus/blob/29a1c656ee7f74b6eba8ce95fa9c239b2f328813/packages/perseus/src/widgets/matcher/matcher.tsx#L164-L172) when the widget renders the options. 

I'm inclined to extract that logic out to a helper for matcher and use it here for now. This means that the rubric type and this public options type will have a property that is named the same, but is conceptually different (one defines the correct answer, the other is the initial render state).

Reply:
It looks like the shuffle and seededRNG utils are part of the Perseus package. I don't think we want to use part of the Perseus package in Perseus Core, so I'm not sure I'll be able to pull it out to a helper function without copying a lot of logic. I might be missing something though.

Reply:
Extracting the `seededRNG()` and `shuffle()` functions to `perseus-core` would be fine. I think we'll find more and more of our "core logic" moving into `perseus-core` and `perseus` becoming more and more a UI-only library.